File: Solutions/2418- Sort the People/Solution.java
class Solution {
    public String[] sortPeople(String[] names, int[] heights) {
        HashMap<Integer, String> myMap = new HashMap<>();
        for (int i = 0; i < names.length; i++) {
            myMap.put(heights[i], names[i]);
        }

        Arrays.sort(heights);
        String[] answer = new String[names.length];
        int counter = 0; 
        for (int i = heights.length - 1; i >= 0; i--) {
            answer[counter] = myMap.get(heights[i]);
            counter++;
        }
        
        return answer;
    }
}

File: Solutions/2418- Sort the People/Solution.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        my_dict = dict()
        for i in range(0, len(names)):
            my_dict[heights[i]] = names[i]

        heights.sort(reverse=True)
        answer = list()
        for i in heights:
            answer.append(my_dict[i])
        
        return answer